modules/debts_api/app/sidekiq/debts_api/v0/form5655/send_confirmation_email_job.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'debts_api/v0/financial_status_report_service'
+
+module DebtsApi
+  class V0::Form5655::SendConfirmationEmailJob
+    include Sidekiq::Job
+
+    sidekiq_retries_exhausted do |job, ex|
```

Comment:
Since you don't have `sidekiq_options retry:` set to any number, this means it will use the default–25. Is that desired?

Reply:
Good suggestion, 25 seems like too much. I've made it 5.